src/ServiceBus.AttachmentPlugin.Tests/When_sending_message_with_sas_uri.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Text;
+
+namespace ServiceBus.AttachmentPlugin.Tests
+{
+    using System.Threading.Tasks;
+    using Microsoft.Azure.ServiceBus;
+    using Xunit;
+
+    public class When_sending_message_with_sas_uri : IClassFixture<AzureStorageEmulatorFixture>
+    {
+        [Fact]
+        public async Task Should_contain_no_sas_uri_if_not_specified()
```

Comment:
Reads funny 😃 `When_sending_message_with_sas_uri.Should_contain_no_sas_uri_if_not_specified`
Perhaps should be located in the other test `When_sending_message`?